<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>949</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>武汉理工大学智能床垫监控系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="scr.qrc">
    <normaloff>:/pix/WHUT.jpg</normaloff>:/pix/WHUT.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>161</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(208, 136, 52);</string>
      </property>
      <property name="text">
       <string>参数设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 157, 235, 255), stop:1 rgba(255, 255, 255,255));
</string>
      </property>
      <item>
       <property name="text">
        <string>COM1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM8</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 157, 235, 255), stop:1 rgba(255, 255, 255,255));
</string>
      </property>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 157, 235, 255), stop:1 rgba(255, 255, 255,255));
</string>
      </property>
      <property name="text">
       <string>调零</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 157, 235, 255), stop:1 rgba(255, 255, 255,255));
</string>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 157, 235, 255), stop:1 rgba(255, 255, 255,255));
</string>
      </property>
      <property name="text">
       <string>断开</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_13" native="true">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(208, 136, 52);</string>
      </property>
      <property name="text">
       <string>功能区</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 157, 235, 255), stop:1 rgba(255, 255, 255,255));
</string>
      </property>
      <property name="text">
       <string>导出8通道数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 157, 235, 255), stop:1 rgba(255, 255, 255,255));
</string>
      </property>
      <property name="text">
       <string>导出睡姿</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>翻身次数</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>12</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>起夜次数</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_14" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>161</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(208, 136, 52);</string>
      </property>
      <property name="text">
       <string>用户信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_12" native="true">
      <property name="styleSheet">
       <string notr="true">border:0px solid black;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
</string>
         </property>
         <property name="text">
          <string>姓名：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
</string>
         </property>
         <property name="text">
          <string>身高：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
</string>
         </property>
         <property name="text">
          <string>年龄：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
</string>
         </property>
         <property name="text">
          <string>体重：</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 157, 235, 255), stop:1 rgba(255, 255, 255,255));
</string>
      </property>
      <property name="text">
       <string>载入用户</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="scr.qrc">:/pix/WHUT.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>20</y>
     <width>139</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(208, 136, 52);</string>
   </property>
   <property name="text">
    <string>当前睡姿</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>1241</width>
     <height>721</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>A</string>
    </attribute>
    <widget class="QChartView" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid black;

</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>20</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_8" native="true">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>220</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>220</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>220</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>139</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(208, 136, 52);</string>
     </property>
     <property name="text">
      <string>活动日志</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>1181</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;\n</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>B</string>
    </attribute>
    <widget class="QChartView" name="widget_f" native="true">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>220</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_d" native="true">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>20</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_a" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid black;

</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_g" native="true">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>220</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_e" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_h" native="true">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>220</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_b" native="true">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QChartView" name="widget_c" native="true">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>139</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(208, 136, 52);</string>
     </property>
     <property name="text">
      <string>活动日志</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>1181</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;\n</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>481</width>
     <height>191</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>1</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>491</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>2</string>
    </attribute>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>491</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>label_11</zorder>
  <zorder>widget_14</zorder>
  <zorder>widget_9</zorder>
  <zorder>widget_13</zorder>
  <zorder>label_10</zorder>
  <zorder>tabWidget</zorder>
  <zorder>tabWidget_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="scr.qrc"/>
 </resources>
 <connections/>
</ui>
